### PR TITLE
docs(samples): Add get bucket metadata sample

### PR DESCRIPTION
**Pull request by frankyn**
cc: @JesseLovelace 
- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)

### CODE REVIEW

File: samples/buckets.js [hunk 1]
```diff
@@ -67,6 +67,40 @@ async function listBuckets() {
   // [END storage_list_buckets]
 }
 
+async function getBucketMetadata(bucketName) {
```
Comment (JustinBeckwith) [inline samples/buckets.js]: Could I trouble you to put this sample in its own file?  Then you can add metadata that gets picked up automatically.  See https://github.com/googleapis/nodejs-bigquery/blob/master/samples/getModel.js#L19 for an example!

File: samples/buckets.js [hunk 2]
```diff
@@ -67,6 +67,40 @@ async function listBuckets() {
...
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+
+  // Get Bucket Metadata
+  const [metadata] = await storage.bucket(bucketName).getMetadata();
+
+  console.log(`BucketName: ${metadata.name}`);
```
Comment (JustinBeckwith) [inline samples/buckets.js]: Could we really simplify this? 

```js
for (let [key, value] of Object.entries(metadata)) {
  console.log(`${key}: ${value}`);
}
```

File: samples/buckets.js (hunk 1 above)
Comment (frankyn) [reply samples/buckets.js]: Sure thing! Thanks for the example.

File: samples/buckets.js (hunk 2 above)
Comment (frankyn) [reply samples/buckets.js]: YES!! Thanks for the tip.

**Pull request merged**